tests/dummy/app/routes/nodes/detail/draft-registrations/index.js

```diff
@@ -8,6 +8,8 @@ export default Ember.Route.extend({
     },
     setupController(controller, model) {
         this._super(controller, model);
+        var metaschemas = this.store.findAll('metaschema');
+        controller.set('metaschemas', metaschemas);
```

Comment:
Agreed- drafts are in no way a blocker for this PR, and wouldn't be even if the API was behaving better. 

Just making the note while the code was fresh.